app/src/test/java/invoice_automation/module/QuickBooksModuleTest.java

```diff
@@ -191,5 +208,30 @@ public void testGetInvoiceWithMatchingMemo_noMatchingInvoice() throws Exception
         // Verify
         assertNull(returnedInvoice);
     }
+
+    @Test
+    public void testUpdateCustomerFromSchool_happyPath() throws Exception {
+        // Setup
+        setupMethodTests();
+        setupSchool();
+
+        // Run
+        Customer returnedCustomer = quickBooksModule.updateCustomerFromSchool(school);
+
+        // Verify
+        assertEquals(school.getSchoolName(), returnedCustomer.getDisplayName());
```

Comment:
create a customer with the expected attributes (either by calling set...() repeatedly or mocking calls to get...()) and do assertEquals(expectedCustomer, returnedCustomer). Customer overloads equals() to check fields rather than just identity so assertEquals() will do the same